examples/oidc_cli/src/main.rs

```diff
@@ -330,9 +355,16 @@ impl OidcCli {
                 Some("sessions") => {
                     self.account(Some(OidcAccountManagementAction::SessionsList));
                 }
-                Some("watch") => {
-                    self.watch().await?;
-                }
+                Some("watch") => match args.next() {
+                    Some(sub) => {
+                        if sub == "--sliding" || sub == "sliding" {
+                            self.sliding_sync().await?;
+                        } else {
+                            println!("unknown subcommand for watch: available is --sliding");
```

Comment:
Nitpick: the help says the subcommand is `sliding`, this says it's `--sliding`. We should stick to one or the other